### PR TITLE
Linting Calendar and EventListSideBar

### PR DESCRIPTION
**Pull request by patrickb42**
# Short Description

simple linting of a file

## Rationale/purpose of the feature

code cleanup

## Link to relevant User Story

none

## Optional: Screenshot(s) of intended state of feature, wire-frame(s), flowchart

Optional attachments here

## Checklist for author and reviewers

- [ ] Commits are well named and descriptive
- [ ] PR is focused on a single problem, is small enough and easy to review
- [ ] PR requests were sent to multiple reviewers
- [ ] PR is reviewed by more than one team member
- [ ] Code is consistently formatted
- [ ] Code is suitably commented on
- [ ] There is no duplicate code
- [ ] Code tries to follow best practices
- [ ] Decide with reviewers if this PR should have tests before merging

### CODE REVIEW

File: src/components/Calendar/Calendar.js [hunk 1]
```diff
@@ -27,10 +40,8 @@ const localizer = dateFnsLocalizer({
   locales,
 });
 
-//drag and drop
```
Comment (petevs) [inline src/components/Calendar/Calendar.js:30]: I wonder if we should leave this comment in and maybe even expand on it. Something along the lines of, "drag and drop not currently implemented due to bug"

**Review by petevs — APPROVED** [review]
Looks good! Thanks for cleaning things up!

File: src/components/Calendar/Calendar.js (hunk 1 above)
Comment (DanielleKoduru) [reply src/components/Calendar/Calendar.js:30]: I agree with Pete, I think leaving a comment regarding the drag and drop here would be helpful for future developers.

**Review by DanielleKoduru — COMMENTED** [review]
This looks great Patrick. Great work!

**Review by DanielleKoduru — APPROVED** [review]

**Pull request merged**